test/images/porter/Makefile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 The Kubernetes Authors.
+# Copyright 2017 The Kubernetes Authors.
```

Comment:
This happened because initially I removed the Makefiles and then created bootstrap and moved them back to Makefile, so resulted in copyright with new year.. :'( I'll take care this..